### PR TITLE
doc: anti-scraper bot troubleshooting

### PR DESCRIPTION
**Pull request by db0**
Adds some info on how to combat scraping bots which can cause site issues, especially on smaller sites.

### CODE REVIEW

File: src/administration/troubleshooting.md [hunk 1]
```diff
@@ -218,3 +218,34 @@ Each of these folders contains a `down.sql` file. We need to run that against ou
...
+```bash
+docker-compose logs --tail=10000 proxy | 
+    grep -o '"[^"]*"$' |     # Extract the last quoted string (user agent)
+    grep -v '^"$' |          # Remove empty quotes
+    tr -d '"' |              # Remove the quotes
+    sort |                   # Sort the user agents
+    uniq -c |               # Count unique occurrences
+    sort -rn |              # Sort numerically in reverse order
+    head -n 10              # Show top 10 results
+```
```
Comment (Nutomic) [inline src/administration/troubleshooting.md:249]: I would put this part first so people can block only the crawlers which are causing them problems.

File: src/administration/troubleshooting.md [hunk 2]
```diff
@@ -218,3 +218,34 @@ Each of these folders contains a `down.sql` file. We need to run that against ou
...
+
+## UI randomly slow or offline
+
+If you notice that your lemmy-ui sometimes becomes sluggish or unresponsive over a period of minutes/hours and then it passes, you might be getting targetted by scraper bots.
+
+There's a lot of scraper bots online and they can easily overwhelm your site when they're behaving too "greedily". 
+
+Unfortunately the existing lemmy-ui has a habit of falling over when polled too eagerly, while the backend still continues to work.
+
+One solution to deal with the scraping bots is to block their user agents. To do so, you can modify your nging_internal.conf to block some of the usual suspects, with this line under `server`
```
Comment (Nutomic) [inline src/administration/troubleshooting.md:230]:
```suggestion
One solution to deal with the scraping bots is to block their user agents. To do so, you can modify your nginx_internal.conf to block some of the usual suspects, with this line under `server`
```

File: src/administration/troubleshooting.md [hunk 3]
```diff
@@ -218,3 +218,34 @@ Each of these folders contains a `down.sql` file. We need to run that against ou
...
...ominfoBot|YaK|VelenPublicWebCrawler|SentiBot|Vagabondo|SEOkicks|SEOkicks-Robot|mtbot/1.1.0i|SeznamBot|DotBot|Cliqzbot|coccocbot|Scrap|SiteCheck-sitecrawl|MauiBot|GumGum|Clickagy|AspiegelBot|Yandex|TkBot|CCBot|Qwantify|MBCrawler|serpstatbot|AwarioSmartBot|Semantici|ScholarBot|proximic|MojeekBot|GrapeshotCrawler|IAScrawler|linkdexbot|contxbot|PlurkBot|PaperLiBot|BomboraBot|Leikibot|weborama-fetcher|NTENTbot|Screaming Frog SEO Spider|admantx-usaspb|Eyeotabot|VoluumDSP-content-bot|SirdataBot|adbeat_bot|TTD-Content|admantx|Nimbostratus-Bot|Mail.RU_Bot|Quantcastboti|Onespot-ScraperBot|Taboolabot|Baidu|Jobboerse|VoilaBot|Sogou|Jyxobot|Exabot|ZGrab|Proximi|Sosospider|Accoona|aiHitBot|Genieo|BecomeBot|ConveraCrawler|NerdyBot|OutclicksBot|findlinks|JikeSpider|Gigabot|CatchBot|Huaweisymantecspider|Offline Explorer|SiteSnagger|TeleportPro|WebCopier|WebReaper|WebStripper|WebZIP|Xaldon_WebSpider|BackDoorBot|AITCSRoboti|Arachnophilia|BackRub|BlowFishi|perl|CherryPicker|CyberSpyder|EmailCollector|Foobot|GetURL|HTTrack|LinkScan|Openbot|Snooper|SuperBot|URLSpiderPro|MAZBot|EchoboxBot|SerendeputyBot|LivelapBot|linkfluence.com|TweetmemeBot|LinkisBot|CrowdTanglebot|Amazonbot|ClaudeBot") { return 444; }
```
Comment (Nutomic) [inline src/administration/troubleshooting.md:233]: Dont include this whole list, but only two or three as examples how the config works. Then you can link an external site for the full list.

File: src/administration/troubleshooting.md [hunk 4]
```diff
@@ -218,3 +218,34 @@ Each of these folders contains a `down.sql` file. We need to run that against ou
...
+```
+
+This is blocking of of the well-known scrapers, but there can be more.
```
Comment (Nutomic) [inline src/administration/troubleshooting.md:236]:
```suggestion
This is blocking of the well-known scrapers, but there can be more.
```

File: src/administration/troubleshooting.md (hunk 3 above)
Comment (db0) [reply src/administration/troubleshooting.md:233]: I don't quite have an external site for such a purpose o.O what would you suggest?

Comment (dessalines) [reply src/administration/troubleshooting.md]: This seems to be fairly well maintained: https://github.com/monperrus/crawler-user-agents/blob/master/crawler-user-agents.json

**Review by Nutomic — APPROVED** [review]

**Pull request merged**